eoscostarica/gGoods
# fix(backend): move password hash creation to backend for user

### Remove security vulnerabilities

### What does this PR do?
Move the password creation and password comparison to the backend side

- Resolve #124 

### Steps to test
1. Sign up as a user or organization
2. Login as a user or organization
3. Check that you can login correctly

#### CheckList
- [x] User password creation
- [x] User password verification (Login)
- [x] User password recovery
- [x] Organization password creation
- [x] Organization password verification (Login)
- [x] Organization password recovery
